azure-pipelines.yml

```diff
@@ -130,22 +134,218 @@ jobs:
+
+  - task: CopyFiles@2
+    displayName: 'Copy Microsoft.Management.Configuration.UnitTests binaries'
+    inputs:
+      SourceFolder: '$(buildOutDir)\Microsoft.Management.Configuration.UnitTests'
+      TargetFolder: '$(artifactsDir)\Microsoft.Management.Configuration.UnitTests'
+
+  - task: CopyFiles@2
+    displayName: 'Copy Microsoft.Management.Configuration.OutOfProc binaries'
+    inputs:
+      SourceFolder: '$(buildOutDir)\Microsoft.Management.Configuration.OutOfProc'
+      TargetFolder: '$(artifactsDir)\Microsoft.Management.Configuration.OutOfProc'
+
+  - task: PublishPipelineArtifact@1
+    displayName: Publish Pipeline Artifacts
+    inputs:
+      targetPath: '$(artifactsDir)'
+      artifact: 'Build.$(artifactIdentifier)'
+
+  - task: ComponentGovernanceComponentDetection@0
+    displayName: Component Governance
+    inputs:
+      scanType: 'Register'
+      verbosity: 'Verbose'
+      alertWarningLevel: 'High'
+
+  # Run BimSkim for all the binaries
+  - task: BinSkim@4
+    displayName: 'Run BinSkim '
+    inputs:
+      arguments: 'analyze
+        "$(buildOutDir)\AppInstallerCLI\winget.exe"
+        "$(buildOutDir)\WinGetUtil\WinGetUtil.dll"
+        "$(buildOutDir)\WindowsPackageManager\WindowsPackageManager.dll"
+        "$(buildOutDir)\Microsoft.Management.Deployment.InProc\Microsoft.Management.Deployment.InProc.dll"
+        "$(Build.SourcesDirectory)\src\WinGetUtilInterop\bin\WinGetUtil*Interop.dll"
+        "$(buildOutDir)\UndockedRegFreeWinRT\winrtact.dll"
+        "$(buildOutDir)\Microsoft.WinGet.Client.Cmdlets\Microsoft.WinGet.Client*.dll"
+        "$(buildOutDir)\ConfigurationRemotingServer\ConfigurationRemoting*Server.dll"
+        "$(buildOutDir)\ConfigurationRemotingServer\ConfigurationRemoting*Server.exe"
+        "$(buildOutDir)\ConfigurationRemotingServer\Microsoft.Management.Configuration*.dll"
+        "$(buildOutDir)\Microsoft.Management.Configuration\Microsoft.Management.Configuration*.dll"
+        "$(buildOutDir)\Microsoft.Management.Configuration.OutOfProc\Microsoft.Management.Configuration*.dll"
+        --config default --recurse'
+
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-publishsecurityanalysislogs.PublishSecurityAnalysisLogs@3
+    displayName: 'Publish Security Analysis Logs'
+
+# Test job runs tests using build artifacts
+
+- job: 'Test'
+  timeoutInMinutes: 120
+  dependsOn: 'Build'
+  condition: succeeded('Build')
+
+  strategy:
+    matrix:
+      x86_release:
+        buildConfiguration: 'Release'
+        buildPlatform: 'x86'
+        testBuildConfiguration: 'TestRelease'
+        artifactIdentifier: 'x86release'
+      x64_release:
+        buildConfiguration: 'Release'
+        buildPlatform: 'x64'
+        testBuildConfiguration: 'TestRelease'
+        artifactIdentifier: 'x64release'
+
+  variables:
+    buildOutDir: $(Pipeline.Workspace)\Build.$(artifactIdentifier)
+    artifactsDir: $(Build.ArtifactStagingDirectory)\$(buildPlatform)

```

Comment:
Why is the build platform needed here?

Reply:
It was before we split the architectures into their own jobs (although it probably wasn't done this way).  Now it isn't, but it also doesn't hurt things.